frontend-zs/.env.sample

```diff
@@ -1,2 +1,3 @@
 REACT_APP_PORT=3001
-REACT_APP_HOST=http://localhost
\ No newline at end of file
+REACT_APP_HOST=http://localhost
+REACT_APP_MAPS_KEY=
```

Comment:
The key is missing from the .env.sample. I get the error: "Google Maps JavaScript API error: InvalidKeyMapError"

Reply:
Yes, I didn't put it because I was afraid someone might use it from outside the seminar groups. I sent in on slack though.